<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesWindow</class>
 <widget class="QDialog" name="PreferencesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="startAtLogin">
     <property name="text">
      <string>Start at login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoUpdateCheck">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Check for updates automatically</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="audioCues">
     <property name="text">
      <string>Enable audio cues</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="cueVolumeLabel">
       <property name="text">
        <string>Cue Vol.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="hotkeyLabel">
       <property name="text">
        <string>Hotkey</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="modeLabel">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="modeUnmute">
       <property name="text">
        <string>Unmute when holding key</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="modeMute">
       <property name="text">
        <string>Mute when holding key</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="cueVolume">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="HotkeyInput" name="hotkey">
       <property name="placeholderText">
        <string>Set a hotkey</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HotkeyInput</class>
   <extends>QLineEdit</extends>
   <header>src/ui/hotkey_input.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
